09-redux/classworks/Agatha Rudko/react/src/pages/home/Home.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState} from 'react';
+import useRedditApi, {withRedditApi} from 'hooks/useRedditApi';
+import {Post} from '../../components/Post/Post';
+import {InfoBar} from "../../components/InfoBar/InfoBar";
+import './Home.css'
+
+
+const Home = () => {
+  const [reactSubreddit, setReactSubreddit] = useState(null);
+  const [about, setAbout] = useState(null);
+  const [fetchReddit] = useRedditApi();
+
+  useEffect(() => {
+    async function fetch() {
+      const data = await fetchReddit(`/${localStorage.getItem('askedSubreddit' || '')}/hot`).then(res => res.json());
+      const about = await fetchReddit(`/${localStorage.getItem('askedSubreddit' || '')}/about`).then(res => res.json());
```

Comment:
you get local storage item twice, but it's quite an expensive operation (you can do it once and then use the value)